packages/web/src/store/sagas.ts

```diff
@@ -165,6 +168,7 @@ export default function* rootSaga() {
     remixSettingsSagas(),
     repostPageSagas(),
     scrollLockSagas(),
+    searchPageSagas(),
```

Comment:
Eh, this could probably be called `searchUserListModalSagas` or something, but all the other user list sagas are called `xxxPageSagas`?